### PR TITLE
Small improvements to the docker lifecycle management document

### PR DESCRIPTION
**Pull request by gurevichmark**
Made small changes to the document to fix some grammar and improve readability.
@zet809 and @whowutwut please review and let me know if you agree with changes.

### CODE REVIEW

File: docs/source/advanced/docker/lifecycle_management.rst [hunk 1]
```diff
@@ -67,11 +67,11 @@ The osimage for dockerhost will be like this ::
 
 Preparing config bridge for dockerhost
 ``````````````````````````````````````
-Currently, the script *xHRM* can be used to config bridge based on a network adapter. In can be replaced by *confignetworks* in the future. To have the dockerhost configure bridge during OS provisioning. The command below can be used: ::
+Currently, the script *xHRM* can be used to config bridge based on a network adapter. In can be replaced by *confignetworks* in the future. To have the dockerhost configure bridge during OS provisioning, use the command: ::
 
  chdef host01 -p postbootscripts="xHRM bridgeprereq eth0:mydocker0"
 
-The script above means configure bridge "mydocker0" based on "eth0". Please **don't modify bridge name "mydocker0"** since it will be used when setting up docker service on dockerhost.
+The above command configures bridge *mydocker0* based on *eth0*. Please **don't modify bridge name "mydocker0"** since it will be used when setting up docker service on dockerhost.
```
Comment (zet809) [inline docs/source/advanced/docker/lifecycle_management.rst]: I forget to mention that the *eth0* may need to modify to the nic name accordingly.
I think you can add it here.

**Comment by zet809** [comment]
@gurevichmark Thanks a lot. The modification looks great to me.

**Comment by gurevichmark** [comment]
Added a sentence about replacing eth0 with nic name.

**Comment by whowutwut** [comment]
Will merge in the changes

**Pull request merged**